docs/javascript-troubleshooting.md

```diff
@@ -27,3 +27,48 @@ You can find further information in [Scope Javascript Agent - Running tests insi
 If you are using certain libraries or version libraries that are not officially supported by the Scope Javascript agent, you might have not see data in Scope after executing the test phase.
 
 You can find further information in [Scope Javascript Agent - Compatibility](javascript-compatibility.md) section.
+
+## I can't see the Test Code in the Test Code Tab
+
+This is most likely caused by a problem with [`SCOPE_SOURCE_ROOT`](javascript-installation.md#environment-variables). While we try to autodetect the root of your repository, it is not always possible. For example, if your folder structure looks like this:
+
+```
+/source/root
+  /src
+    /ui
+      cypress.json
+      /cypress
+        /integration
+          integration.spec.js
+    /admin-ui
+      cypress.json
+      /cypress
+        /smoke
+          smoke.spec.js
```

Comment:
not sure if this is the best way to represent a folder tree structure

Reply:
Have you tried using a tool? E.g: https://www.npmjs.com/package/mddir
More info about it in [this post](https://stackoverflow.com/questions/19699059/representing-directory-file-structure-in-markdown-syntax)

Reply:
cool! Thanks!

Reply:
tried 👍 , much better

Reply:
awesome